File: CoddyTech/C for Beginners/decision making.c
#include <stdio.h>

int main() {
    int a = 3;
    int b = 2;

    if (a >= b) {
        printf("You are here!");
    }

    int age;
    scanf("%d", &age);
    if (age > 18) {
        printf("Above 18");
    }
    else {
        printf("Below or equal 18");
    }

    for (int i = 1; i <= 10; i++) {
        printf("hello\n");
    }

    
    return 0;
};

File: CoddyTech/Java Fundamentals/src/decisionmaking.java
import java.util.Scanner;

public class decisionmaking {
    public static void main(String[] args) {
        int a = 9;
        int b = 9;

        // Don't change below this line
        if (a >= b) {
            System.out.println("You are here!");
        }

        Scanner scanner = new Scanner(System.in);
        int age = scanner.nextInt();
        // Don't touch above this line
        // Write code here
        if (age > 18) {
            System.out.println("Above 18");
        }
        if (age <= 18) {
            System.out.println("Below or equal 18");
        }
    }
}
